src/raftstore/store/store.rs

```diff
@@ -1693,34 +1676,13 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         for peer in self.region_peers.values_mut() {
             peer.peer_stat.last_written_bytes = peer.peer_stat.written_bytes;
```

Comment:
can we save the last_written_bytes/keys in PD worker and update in the heartbeat msg?